src/odemis/gui/comp/miccanvas.py

```diff
@@ -1084,6 +1087,10 @@ class SparcARCanvas(DblMicroscopeCanvas):
     def __init__(self, *args, **kwargs):
         super(SparcARCanvas, self).__init__(*args, **kwargs)
         self.abilities -= {CAN_ZOOM, CAN_DRAG}
+
+        # Disable DC mode tool
+        self.allowed_modes = guimodel.ALL_TOOL_MODES.difference((guimodel.TOOL_RO_ANCHOR,))
```

Comment:
Let's also forbid ROA and SPOT. =>
`guimodel.ALL_TOOL_MODES - {guimodel.TOOL_SPOT, guimodel.TOOL_ROA, guimodel.TOOL_RO_ANCHOR}`

What I don't really get though is that in `tabs.SparcAcquisitionTab`, line 917, there was already a line supposed to handle this:
```
        # Don't show ROA/spot on the AR view
        panel.vp_sparc_tr.canvas.allowed_modes = {TOOL_NONE}
```
Please, have a look at why that line didn't do the right thing, and drop it or fix the original issue.